src/app/inventory/d2-stores.ts

```diff
@@ -449,27 +443,44 @@ function updateBasePower(
   allItems: DimItem[],
   store: DimStore,
   defs: D2ManifestDefinitions,
-  hasClassified: boolean,
   characterProgress: DestinyCharacterProgressionComponent | undefined,
   bonusPowerProgressionHash: number | undefined
 ) {
   if (!store.isVault) {
     const def = defs.Stat.get(StatHashes.Power);
     const { equippable, unrestricted } = maxLightItemSet(allItems, store);
+
+    // ALL WEAPONS count toward your drops. armor on another character doesn't count.
+    // (maybe just because it's on a different class? who knows. can't test.)
+    const dropPowerItemSet = maxLightItemSet(
+      allItems.filter((i) => i.bucket.inWeapons || i.owner === 'vault' || i.owner === store.id),
+      store
+    ).unrestricted;
+    const dropPowerLevel = getLight(store, dropPowerItemSet);
+
     const unrestrictedMaxGearPower = getLight(store, unrestricted);
     const unrestrictedPowerFloor = Math.floor(unrestrictedMaxGearPower);
     const equippableMaxGearPower = getLight(store, equippable);
 
-    const differentEquippableMaxGearPower =
-      (unrestrictedMaxGearPower !== equippableMaxGearPower && equippableMaxGearPower) || undefined;
+    const statProblems: DimCharacterStat['statProblems'] = {};
+
+    statProblems.notEquippable = unrestrictedMaxGearPower !== equippableMaxGearPower;
+    statProblems.notOnStore = dropPowerLevel !== unrestrictedMaxGearPower;
+
+    statProblems.hasClassified = allItems.some(
+      (i) =>
+        i.classified &&
+        (i.location.inWeapons ||
+          i.location.inArmor ||
+          (isD1Item(i) && i.bucket.hash === BucketHashes.Ghost))
```

Comment:
Maybe we can just check for power?

Reply:
like i.power? sure yeah

Reply:
I *think* that distinguishes between actual power and like... sparrow speed. I'm not sure.